beets/ui/commands.py

```diff
@@ -801,6 +830,53 @@ def should_resume(self, path):
                            "was interrupted. Resume (Y/n)?"
                            .format(displayable_path(path)))
 
+    def _get_plugin_options(self, task):
```

Comment:
Of course, I now notice that I suggested the name with `choices`. :flushed: Sorry!